databricks_cli/workspace/api.py

```diff
@@ -83,6 +83,9 @@ def __init__(self, api_client):
     def get_status(self, workspace_path):
         return WorkspaceFileInfo.from_json(self.client.get_status(workspace_path))
 
+    def get_status_json(self, workspace_path):
+        return self.client.get_status(workspace_path)
```

Comment:
similar to dbfs, we can just define this inside stack CLI. This interface is too similar to get_status and it will cause confusions.